psydac/linalg/solvers.py

```diff
@@ -62,19 +60,25 @@ def inverse(A, solver, **kwargs):
     # `InverseLinearOperator` subclass in this module:
     solvers_dict = {
         'cg'       : ConjugateGradient,
-        'pcg'      : PConjugateGradient,
         'bicg'     : BiConjugateGradient,
         'bicgstab' : BiConjugateGradientStabilized,
-        'pbicgstab': PBiConjugateGradientStabilized,
         'minres'   : MinimumResidual,
         'lsmr'     : LSMR,
         'gmres'    : GMRES,
     }
 
+    # Only these solvers accept a preconditioner argument for now:
+    solvers_with_pc = ['cg', 'bicgstab']
+
     # Check solver input
     if solver not in solvers_dict:
         raise ValueError(f"Required solver '{solver}' not understood.")
```

Comment:
In order to allow for the capitalized solver names, here we should convert the input string to lower case:

```suggestion
    # Only these solvers accept a preconditioner argument for now:
    solvers_with_pc = ['cg', 'bicgstab']

    # Convert input solver string to lower case
    solver = solver.lower()

    # Check solver input
    if solver not in solvers_dict:
        raise ValueError(f"Required solver '{solver}' not understood.")
```